src/test/java/com/artipie/PromuSliceITCase.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * The MIT License (MIT) Copyright (c) 2020-2021 artipie.com
+ * https://github.com/artipie/artipie/LICENSE.txt
+ */
+package com.artipie;
+
+import com.amihaiemil.eoyaml.Yaml;
+import com.amihaiemil.eoyaml.YamlMappingBuilder;
+import com.artipie.http.client.jetty.JettyClientSlices;
+import com.artipie.http.headers.Accept;
+import com.artipie.http.rs.RsStatus;
+import io.vertx.reactivex.core.Vertx;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import javax.ws.rs.HttpMethod;
+import javax.ws.rs.core.MediaType;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * IT tests for Prometheus metrics.
+ * @since 0.23
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+final class PromuSliceITCase {
+
+    @Test
+    void givesAccessWhenPrometheusConfigIsAvailable(
+        @TempDir final Path root) throws Exception {
+        final int port = runServer(
+            root,
+            Yaml.createYamlMappingBuilder()
+                .add(
+                    "metrics",
+                    Yaml.createYamlMappingBuilder()
+                        .add("type", "prometheus")
+                        .build()
+                )
+        );
+        final HttpURLConnection con = (HttpURLConnection) new URL(formatUrl(port))
+            .openConnection();
```

Comment:
Does not `HttpURLConnection` implement closable? If so, I'd suggest to use `try-finally` block here.

Reply:
I'm afraid not.